myhpi/static/js/admin/easymde_custom.js

```diff
@@ -157,3 +163,16 @@ function addQuorum(editor){
     output = "\n|quorum|(/)";
     cm.replaceSelection(output);
 }
+
+function addResolution(editor){
+    const cm = editor.codemirror;
+    const position = cm.getCursor();
+
+    const output = " **[//]** (HHT: ).";
```

Comment:
You need to ouput | (pipe), otherwise the resolution will not be highlighted. I would also prefer if this button also outputted the actual resolution text ("Wir beschließen bis zu x€ für y auszugeben"

Reply:
when rendering the markdown to html, blocks with something like [1|2|3] will be highlighted, that was what I orignally meant. 

the other thing is how the content of the commands is actually stored. All the other buttons just output smth like `|start|(15:24)` to reduce duplication of the actual text snippets which people could mistype. A markdown processor (have a look at extensions.py) then takes this snippet and replaces it with the actual Text ("Sitzungsbeginn 15:24") in the correct locale. You could implement the resolution in the same way (e.g. `|resolution|(ding)(betrag)[1|2|3]`)

Reply:
we will do another release as soon as your PRs are merged :)